iam/keycloak/README.md

```diff
@@ -0,0 +1,14 @@
+# Keycloak for dojot
+
+The [Keycloak](https://www.keycloak.org/) is used by the dojot platform as an
+_Identity and Access Manager_ ([IAM](https://en.wikipedia.org/wiki/Identity_management)).
+
+
+In addition to generating access tokens for the platform's APIs, the Keycloak
+has also been customized to run a specific [provider](https://www.keycloak.org/docs/latest/server_development/index.html#_providers)
```

Comment:
Maybe change to `https://www.keycloak.org/docs/12.0/server_development/index.html` or to quote the version used in the moment.

Reply:
Good idea!